src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,20 @@
+import Item from "./ItemList";
+import "./todolist.scss";
+
+export type Todolist = {
+  text: string;
+};
+
+type Props = {
+  ItemList: Todolist[];

```

Comment:
```suggestion
  ItemList: ToDo[];
```

Let's go with `ToDo` and avoid the postfix `fix` since `Todolist` it's not a list but an item inside of the array.